src/pages/commits/ActiveCommits/CommitChartLabel.test.tsx

```diff
@@ -0,0 +1,81 @@
+import userEvent from "@testing-library/user-event";
+import { render, act } from "test_utils/test-utils";
+import { CommitChartLabel } from "./CommitChartLabel";
+
+const RenderCommitChartLabel = (version) => (
+  <CommitChartLabel
+    githash={version.revision.substring(0, 5)}
+    createTime={version.createTime}
+    author={version.author}
+    message={version.message}
+  />
+);
+
+afterEach(() => {
+  jest.clearAllTimers();
+  jest.clearAllMocks();
+});
+describe("CommitChartLabel", () => {
+  test("Displays author, githash and createTime", () => {
+    const { queryByDataCy } = render(RenderCommitChartLabel(versionShort));
+    expect(queryByDataCy("commit-label")).toHaveTextContent(
+      "4137c 6/16/21 11:38 PMMohamed Khelif"
+    );
+  });
+
+  test(
+    "Displays shortened commit message and the 'more' button if commit message" +
+      "is too long",
+    async () => {
+      const { queryByDataCy } = render(RenderCommitChartLabel(versionLong));
+      expect(queryByDataCy("tooltip-button")).toBeInTheDocument();
+      expect(queryByDataCy("commit-label")).toHaveTextContent(
+        "4137c 6/16/21 11:38 PMMohamed Khelif -SERVER-57332 Create skeleton InternalDo...more"
+      );
+    }
+  );
+
+  test("Displays entire commit message if it does not break length limit", async () => {
+    const { queryByDataCy } = render(RenderCommitChartLabel(versionShort));
+    expect(queryByDataCy("commit-label")).toHaveTextContent(
+      "SERVER-57332 Create skeleton Internal"
+    );
+  });
+
+  test(
+    "Clicking on the 'more' button should open a tooltip that displays" +
+      "the long commit message",
+    async () => {
```

Comment:
do we get the same result after removing async. I don't think we need it here since we don't use await in the function body